jobs/rocoto/waveinit.sh

```diff
@@ -5,8 +5,7 @@ source "${HOMEgfs}/ush/preamble.sh"
 ###############################################################
 echo
 echo "=============== START TO SOURCE FV3GFS WORKFLOW MODULES ==============="
-#. ${HOMEgfs}/ush/load_fv3gfs_modules.sh
-. ${HOMEgfs}/ush/load_ufswm_modules.sh
+. ${HOMEgfs}/ush/load_fv3gfs_modules.sh
```

Comment:
We may still want to use the ufs modulefile. @aerorahul?

Reply:
I thought I had chatted offline with @aerorahul about reverting this back. I'm good either way, let me know if we should switch this back.

Reply:
Also note, I did not change the fcst job, just the wave pre/post jobs.

Reply:
I am fine either way.  I will leave it to @JessicaMeixner-NOAA

Reply:
Pro of ufs-weather-model modules: this is what it's built with, ensures consistency as things might get out of sync while developing.  However, I thought we were all supposed to be moving towards using the same modules, which I thought was the fv3gfs_modules so I thought that was the goal?

Reply:
It is the goal. Yes. For ops, it is a requirement.
However, these are the rocoto wrappers used in development. The flexibility to allow for asynchronous updates to the components would be desirable.

Reply:
Okay, let me switch these back then.